### PR TITLE
add lines changed and removed in gitstatus

### PR DESCRIPTION
**Pull request by hemna**
This patch adds the number of lines added and removed as part
of the gitstatus line.

### CODE REVIEW

**Review by gforsyth — CHANGES_REQUESTED** [review]
Hey @hemna -- thanks for putting this in!  I think CI is mostly unhappy with the formatting of the news file.

File: xonsh/prompt/gitstatus.py [hunk 1]
```diff
@@ -93,6 +97,29 @@ def _get_tag_or_hash():
     return tag_or_hash if have_tag_name else _get_def("HASH") + hash_
 
 
+def _get_files_changed():
+    try:
+        changed = _check_output(["git", "diff", "--numstat"])
+    except subprocess.CalledProcessError as e:
+        return
```
Comment (gforsyth) [inline xonsh/prompt/gitstatus.py:104]:
```suggestion
    except subprocess.CalledProcessError:
        return
```

`flake8` is unhappy about the unused `e` variable

Comment (hemna) [reply xonsh/prompt/gitstatus.py:104]: How do you run all the tests locally?   Usually I have tox.ini for all my python libs/projects and can simply run tox -e<env>

Comment (hemna) [reply xonsh/prompt/gitstatus.py]: I found the run-tests.xsh script and used that to verify flake8 locally

**Review by gforsyth — APPROVED** [review]
Thanks for putting this in @hemna !  And I see this is your first contribution to `xonsh`.  Welcome!

**Pull request merged**